<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventanas</class>
 <widget class="QMainWindow" name="ventanas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ventanas</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="Game_window" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>571</width>
      <height>481</height>
     </rect>
    </property>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>-10</y>
       <width>370</width>
       <height>600</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>361</width>
       <height>481</height>
      </rect>
     </property>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>300</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>JUGAR</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>300</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>SALIR</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>70</y>
        <width>200</width>
        <height>200</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    background-image: url(:/imagenes/titulo..png);
    background-repeat: no-repeat;
    background-position: center;

}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
